dynamiqs/time_array.py

```diff
@@ -508,19 +524,37 @@ def shape(self) -> tuple[int, ...]:
     def mT(self) -> TimeArray:
         return SummedTimeArray([tarray.mT for tarray in self.timearrays])
 
-    def __call__(self, t: float) -> Array:
-        return jax.tree_util.tree_reduce(
-            jnp.add, [tarray(t) for tarray in self.timearrays]
-        )
-
     def reshape(self, *new_shape: int) -> TimeArray:
         return SummedTimeArray(
             [tarray.reshape(*new_shape) for tarray in self.timearrays]
         )
 
+    def broadcast_to(self, *new_shape: int) -> TimeArray:
+        return SummedTimeArray(
+            jtu.tree_map(
+                lambda x: x.broadcast_to(*new_shape),
+                self.timearrays,
+                is_leaf=lambda x: isinstance(x, TimeArray),
+            )
+        )
+
     def conj(self) -> TimeArray:
         return SummedTimeArray([tarray.conj() for tarray in self.timearrays])
 
+    def in_axes(self) -> PWCTimeArray:
```

Comment:
As already discussed privately, this is bugged for common batching over several timearrays summed together.